### ISSUE TITLE
Bug: Unhandled token error if angular pipe has a parameter

### ISSUE THREAD

**Issue by valera33** [prettier/plugin-pug#70]
## Info

| Tool     | Version       |
| -------- | ------------- |
| Plugin   | v1.1.6        |
| Prettier | v1.19.1       |
| Node     | v13.6.0       |
| OS       | linux |

## Input

```pug
div
  | {{'foo' | baz:bar}}
```

## Output or Error

```bash
Error: Unhandled token: {"type":"text","loc":{"start":{"line":2,"column":5},"end":{"line":2,"column":24}},"val":"{{'foo' | baz:bar}}"}
[error]     at PugPrinter.build (/home/valeri/workspaceA4/notices/node_modules/@prettier/plugin-pug/dist/printer.js:77:23)
[error]     at Object.print (/home/valeri/workspaceA4/notices/node_modules/@prettier/plugin-pug/dist/index.js:63:40)
[error]     at callPluginPrintFunction (/home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:12988:18)
[error]     at /home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:12939:16
[error]     at Object.printComments (/home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:12660:17)
[error]     at printGenerically (/home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:12938:22)
[error]     at printAstToDoc (/home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:12950:13)
[error]     at coreFormat (/home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:13271:13)
[error]     at format (/home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:13516:73)
[error]     at formatWithCursor (/home/valeri/workspaceA4/notices/node_modules/prettier/bin-prettier.js:13532:12)

```

## Expected Output

```pug
div
  | {{'foo' | baz:bar}}
```

**Comment by Shinigami92** [comment]
I will try my best, but let me tell you thats not that easy:

https://github.com/prettier/plugin-pug/blob/34f3a5d3dd9e09864cee17b960ceecb39bc26c48/test/issues/issue-9/formatted.pug#L8

😆

**Comment by Shinigami92** [comment]
This fix needs a bit more brain... will target it on weekend

**Comment by Shinigami92** [comment]
Referencing #9

**Comment by Shinigami92** [comment]
Have released a v1.1.8
Please report me any issues if there are some
Otherwise I can close this issue

**Comment by valera33** [comment]
Looks good, we can close the issue, thanks!
